controller/client/client.go

```diff
@@ -351,6 +435,38 @@ func (c *Client) StreamJobEvents(appID string, output chan *ct.JobEvent) (stream
 	return c.ResumingStream("GET", fmt.Sprintf("/apps/%s/jobs", appID), output)
 }
 
+func (c *Client) WatchJobEvents(appID string) (*JobWatcher, error) {
+	events := make(chan *ct.JobEvent)
+
+	stream, err := c.StreamJobEvents(appID, events)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return newJobWatcher(events, stream), nil
+}
+
+func (c *Client) ExpectedScalingEvents(actual, expected map[string]int, clusterSize int) ct.JobEvents {
```

Comment:
Does this need to be a method on the client? Given it just creates `ct.JobEvents`, maybe it should be in the types package?